### DISCUSSION TITLE
Template formatting changed in 0.34.5

### DISCUSSION THREAD

**Discussion by braincomb** [vuejs/language-tools#1174] [Q&A] [answered]
I noticed template formatting has changed with the new version (0.34.5), here's an example of the diff:

![diff example](https://i.gyazo.com/aced30505f5ad1eef44ca0df80399dd0.png)

The new way seems packed and I prefer the old formatting, any way to change/revert that?

**Comment by johnsoncodehk** [comment, marked as answer]
You could use `@volar-plugins/prettyhtml`, see https://github.com/johnsoncodehk/volar-starter/blob/master/volar.config.js, https://github.com/johnsoncodehk/volar-plugins/tree/master/packages/prettyhtml.

**Comment by izcream** [comment]
I have same issues on custom-component too. on native html element seem format fine. I do follow answer add prettyhtml plugin to my project but its still format issues. any solution?

**Reply by johnsoncodehk** [reply]
emmm what format issues?

**Reply by izcream** [reply]
Oops. Sorry it's too long i forgot about it, now look like my issues solve already on all my vue project .😂
